Resources/config/columns.xml

```diff
@@ -19,48 +19,49 @@
     </parameters>
 
     <services>
-        <service id="grid.column.text" class="%grid.column.text.class%" public="false">
-            <tag name="grid.column.extension" />
```

Comment:
Can you please explain why did you decided to change the tag name from `grid.column.extension` to `apy_grid.column` ?